moninag/static/src/ts/app/servers/server-detail.component.html

```diff
@@ -0,0 +1,12 @@
+<div *ngIf="server" class="form-group">
+
+    <label>Server:</label>
+    <div>Name: {{server.name}} </div>
```

Comment:
Name and other stuff should be inside <label> for further style design enhancement